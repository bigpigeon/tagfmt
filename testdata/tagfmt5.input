//tagfmt -s
package main

type User struct {
	Name     string `json:"name" xml:"name" gorm:"type:varchar(64);unique_index"`
	Password string `json:"password" gorm:"type:varchar(128)" xml:"password"`
    EmptyTag  string
	City string `json:"group" gorm:"type:varchar(64)" xml:"group"`
	State string `json:"state" gorm:"type:varchar(64)" xml:"state"`
}