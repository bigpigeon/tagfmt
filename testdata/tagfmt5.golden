//gofmt -s
package main

type User struct {
	Name     string `gorm:"type:varchar(64);unique_index" json:"name"     xml:"name"`
	Password string `gorm:"type:varchar(128)"             json:"password" xml:"password"`
	EmptyTag string
	City     string `gorm:"type:varchar(64)" json:"group" xml:"group"`
	State    string `gorm:"type:varchar(64)" json:"state" xml:"state"`
}
